### PR TITLE
chore(IT Wallet): [SIW-950] Add ID Pay initiatives cards in new wallet section

### PR DESCRIPTION
**Pull request by mastro993**
> [!WARNING]
> This PR depends on https://github.com/pagopa/io-app/pull/5619

## Short description
Include a summary of the changes.

## List of changes proposed in this pull request
- Added SVG asset for the ID Pay wallet card
- Added ID Pay cards in the wallet section

## How to test
You should use the [io-dev-api-server ](https://github.com/pagopa/io-dev-api-server).
Within the Profile section, enable the **ID Pay** and **New wallet section** feature flags. You should be able to see and interact with ID Pay cards in the wallet section

### CODE REVIEW

File: ts/features/newWallet/components/WalletCardsCategoryContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+/**
+ * This component handles the rendering of cards of a specific category.
+ * The component also handles logic behind card stacking and animations
+ */
+const WalletCardsCategoryContainer = ({
+  label,
+  iconName,
+  cards
+}: WalletCategoryStackContainerProps) => {
+  const isStacked = cards.length > 1;
```
Comment (hevelius) [inline ts/features/newWallet/components/WalletCardsCategoryContainer.tsx:20]: Is this the boolean value to view the cards stacked or unstacked? If I manually change this value to false should I get the cards unstacked?

File: ts/features/newWallet/components/WalletCardsContainer.tsx
Comment (hevelius) [inline ts/features/newWallet/components/WalletCardsContainer.tsx:1]: what do you think to add a test for this component?

**Review by hevelius — CHANGES_REQUESTED** [review]

File: ts/features/newWallet/components/WalletCardsCategoryContainer.tsx
[the remark this answers is not in the document]
Comment (mastro993) [reply ts/features/newWallet/components/WalletCardsCategoryContainer.tsx:1]: Added in https://github.com/pagopa/io-app/pull/5625/commits/18e868402cfd474015326fc215fc5b546b396af5

File: ts/features/newWallet/components/WalletCardsContainer.tsx
Comment (mastro993) [reply ts/features/newWallet/components/WalletCardsContainer.tsx:1]: Added in https://github.com/pagopa/io-app/pull/5625/commits/204df259149d4d7826c2e5aca2d54700e9d7643d

**Review by hevelius — APPROVED** [review]
LGTM. Kudos for the great job.

**Pull request merged**